Atomic/syscontainers.py

```diff
@@ -1355,3 +1361,77 @@ def get_out_checksum(obj): return obj.out_checksum if hasattr(obj, 'out_checksum
+        rootfs = os.path.join(rpm_content, "usr/lib/containers/atomic", name)
+        os.makedirs(rootfs)
+        try:
+            spec_file = os.path.join(temp_dir, "container.spec")
+            self._checkout(repo, name, image, 0, False, destination=rootfs, prefix=rpm_content)
+            labels = {k.lower() : v for k, v in image_inspect.get('Labels', {})}
+            summary = labels.get('summary', name)
+            version = labels.get("version", "1.0")
+            release = labels.get("release", "1.0")
+            license_ = labels.get("license", "GPLv2")
+            url = labels.get("url")
+            source0 = labels.get("source0")
+            requires = labels.get("requires")
+            provides = labels.get("provides")
+            conflicts = labels.get("conflicts")
+            description = labels.get("description")
+
+            self._generate_spec_file(spec_file, rpm_content, name, summary, license_, version=version, release=release,
+                                     url=url, source0=source0, requires=requires, provides=provides, conflicts=conflicts,
+                                     description=description)
+
+            cwd = os.getcwd()
+            cmd = ["rpmbuild", "--noclean", "-bb", spec_file,
+                   "--define", "_sourcedir %s" % temp_dir,
+                   "--define", "_specdir %s" % temp_dir,
+                   "--define", "_builddir %s" % temp_dir,
+                   "--define", "_srcrpmdir %s" % cwd,
+                   "--define", "_rpmdir %s" % cwd,
+                   "--build-in-place",
+                   "--buildroot=%s" % rpm_content]
+            util.write_out(" ".join(cmd))
+            if not self.display:
+                util.check_call(cmd)
+            return False
+        finally:
+            shutil.rmtree(temp_dir)
+
+    def _generate_spec_file(self, out, destdir, name, summary, license_, version="1.0", release="1", url=None,
+                            source0=None, requires=None, conflicts=None, provides=None, description=None):
+        spec = "%global __requires_exclude_from ^.*$\n"
+        spec = spec + "%global __provides_exclude_from ^.*$\n"
+
+        fields = {"Name" : "%s-%s" % (RPM_NAME_PREFIX, name), "Version" : version, "Release" : release, "Summary" : summary,
+                  "License" : license_, "URL" : url, "Source0" : source0, "Requires" : requires,
+                  "Provides" : provides, "Conflicts" : conflicts}
+        for k, v in fields.items():
+            if v is not None:
+                spec = spec + "%s:\t%s\n" % (k, v)
+
+        spec = spec + "\n%description\n"
+        if description:
+            spec = spec + "%s\n" % description
+
+        spec = spec + "\n%files\n"
+        for root, _, files in os.walk(os.path.join(destdir, "etc")):
+            rel_path = os.path.relpath(root, destdir)
+            for f in files:
+                spec += "%config \"%s\"\n" % os.path.join("/", rel_path, f)
```

Comment:
I'm confused, this line looks like you want to install container's /etc to host's /etc.